plover/meta/conditional.py

```diff
@@ -0,0 +1,10 @@
+from plover.formatting import _LookAheadAction
+
+def meta_if_next_matches(ctx, meta):
+    pattern, result1, result2 = meta.split('/')
+    action_list = []
+    for alternative in result1, result2:
+        action = ctx.new_action()
+        action.text = alternative
```

Comment:
Although not really needed for Melani system, this is some suggestions:

* perhaps the alternative should be interpreted as a Plover stroke? so people can do something like `{=pattern1/alternative1/{=pattern2/alternative2/fallback}}`
* or just `{=pattern1/alternative1/pattern2/alternative2/fallback}`. Currently it doesn't do anything useful when given more than 2 slashes anyway.
* currently there's no way to include a slash in the pattern or the replacement.

Reply:
> * perhaps the alternative should be interpreted as a Plover stroke? so people can do something like `{=pattern1/alternative1/{=pattern2/alternative2/fallback}}`
> * or just `{=pattern1/alternative1/pattern2/alternative2/fallback}`. Currently it doesn't do anything useful when given more than 2 slashes anyway.

I really don't want to increase the complexity.

> * currently there's no way to include a slash in the pattern or the replacement.

I've amended the code to support that use case.

> * or should this be included in the orthography rules regex instead? The user might want to define multiple stroke to output the same prefix, and there's cumbersome.

Orthography rules are really limited, they're only used when prefixing a word (`{^post}`), and an initial prefix (`{pre^}`) is not taken into account.